wdio.conf.ts

```diff
@@ -0,0 +1,128 @@
+/// <reference types="wdio-electron-service" />
+import type { Server } from 'http'
+
+declare global {
+  // eslint-disable-next-line no-var
+  var testServer: Server | undefined
+}
+
+const getAppBinaryPath = () => {
+  for (const arg of process.argv) {
+    if (arg.startsWith('--app-path=')) {
+      const path = arg.split('=')[1]
+      console.log('Binary path:', path)
+      return path
+    }
+  }
+  console.log('No binary argument found, using default binary path')
+  return undefined
+}
+
+export const config: WebdriverIO.Config = {
+  runner: 'local',
+  tsConfigPath: './test/tsconfig.json',
+  specs: ['./specs/**/*.ts'],
+  // Patterns to exclude.
+  exclude: [
+    // 'path/to/excluded/files'
+  ],
+  maxInstances: 1,
+  capabilities: [
+    {
+      browserName: 'electron',
+      // Electron service options
+      // see https://webdriver.io/docs/desktop-testing/electron/configuration/#service-options
+      'wdio:electronServiceOptions': {
+        appBinaryPath: getAppBinaryPath(),
+      },
+    },
+  ],
+
+  // Level of logging verbosity: trace | debug | info | warn | error | silent
+  logLevel: 'silent',
+  bail: 0,
+  waitforTimeout: 10000,
+  connectionRetryTimeout: 120000,
+  connectionRetryCount: 3,
+  services: ['electron'],
+  framework: 'mocha',
+  reporters: ['spec'],
+  mochaOpts: {
+    ui: 'bdd',
+    timeout: 60000,
+  },
+
+  onPrepare: async function () {
```

Comment:
Webdriver IO [configuration file](https://webdriver.io/docs/configurationfile/) exposes a few hooks that we can use before (or during) our tests.

Here, I'm using the `onPrepare` hook to setup a test HTTP server locally so we can capture requests from. This avoids external dependency network dependency with `quickpizza`.